validation/src/main/java/io/syndesis/qe/validation/OpenshiftValidationSteps.java

```diff
@@ -169,6 +169,21 @@ public void checkThatPodIsNotRedeployed(String podName) {
             .isFalse();
     }
 
+    @Then("^check that the pod \"([^\"]*)\" (has|has not) appeared$")
```

Comment:
either rename the step or change the implementation, as currently the step is named `pod has appeared`, but it is checking if the `pod is running` (it could appear and immediately fail, in that case it won't be `running`)